### ISSUE TITLE
Support partial results

### ISSUE THREAD

**Issue by knazarov** [tarantool/crud#53]
Add support for getting only a subset of fields. Like this:

```lua

crud.get('customers', 1, {fields={'id', 'name'}})
---
- metadata:
  - {'name': 'id', 'type': 'unsigned'}
  - {'name': 'name', 'type': 'string'}
  rows:
  - [1, 'Elizabeth']
...
```

**Comment by knazarov** [comment]
Note: it is possible that we'd want to keep `bucket_id` and primary key always in the result, no matter if they were specified or not. Let's think about it.

**Comment by dokshina** [comment]
Closed by #116, #117
